flink-connectors/flink-sql-connector-hive-2.3.9/pom.xml

```diff
@@ -115,6 +115,10 @@ under the License.
 									<pattern>com.google</pattern>
 									<shadedPattern>org.apache.flink.hive.shaded.com.google</shadedPattern>
 								</relocation>
+								<relocation>
```

Comment:
I found some other packages that might lead to potential conflicts. Maybe you can relocate them in one shot:
- org.joda.time
- com.fasterxml.jackson

Reply:
Sure.

Reply:
Keep in mind that Jackson is already in `flink-shaded`